Modules/Bridge/NumPy/include/itkPyBuffer.hxx

```diff
@@ -59,9 +59,9 @@ PyBuffer<TImage>::_GetImageViewFromArray(PyObject * arr, PyObject * shape, PyObj
 
   SizeType size;
 
-  if (PyObject_GetBuffer(arr, &pyBuffer, PyBUF_ND | PyBUF_ANY_CONTIGUOUS) == -1)
+  if (PyObject_GetBuffer(arr, &pyBuffer, PyBUF_ANY_CONTIGUOUS) == -1)
```

Comment:
- For the record, Matt (@thewtex) added the `PyBUF_ND` flag with https://github.com/InsightSoftwareConsortium/ITKBridgeNumPy/pull/14, Feb 24, 2017.